products/sle12/profiles/anssi_bp28_intermediary.profile

```diff
@@ -0,0 +1,25 @@
+documentation_complete: true
+
+metadata:
+    SMEs:
+        - abergmann
+
+title: 'ANSSI-BP-028 (intermediary)'
+
+description: |-
+    This profile contains configurations that align to ANSSI-BP-028 v1.2 at the intermediary hardening level.
+
+    ANSSI is the French National Information Security Agency, and stands for Agence nationale de la sécurité des systèmes d'information.
+    ANSSI-BP-028 is a configuration recommendation for GNU/Linux systems.
+
+    A copy of the ANSSI-BP-028 can be found at the ANSSI website:
+    https://www.ssi.gouv.fr/administration/guide/recommandations-de-securite-relatives-a-un-systeme-gnulinux/
+
+    Only the components strictly necessary to the service provided by the system should be installed.
+    Those whose presence can not be justified should be disabled, removed or deleted.
+    Performing a minimal install is a good starting point, but doesn't provide any assurance
+    over any package installed later.
+    Manual review is required to assess if the installed services are minimal.
+
+selections:
+  - anssi_sle:all:intermediary
```

Comment:
Hi @yuumasato , thanks for the review and the comment, because I was just looking at an issue in that context :).

The general idea is once we have the full coverage to drop that file if possible. But at this stage we encounter rules that are for example relevant only for other platforms, and not relevant for SUSE, for one or other reason (i.e. package_tftp-server_removed).  I am not sure, at this stage, how we should handle, one obvious option is to leave the rule in the profile and just do not build it for the SUSE platforms , with the `prodtype` flag.

For now this option seem not very explicit and I am looking for some mechanism to explicitly state that in the profile definition, so it is more obvious why this or this rule is included.
So in that context I would gladly accept any suggestions and will be grateful of your advice on that.

Reply:
@teacup-on-rockingchair I'm not sure I understand the problem correctly. The rules in the control file should only be included in the profile if the `prodtype` is applicable.
So `package_tftp-server_removed` should not be available in the DS and it should not be selected in the profile.
I just built `sle15` and I don't see the rule in the DS nor the profile, do you get a different result?
You can also check the "compiled" profiles in `build/sle15/profiles`, there you'll find final state of the profile generated from the controls.

I guess initial the idea with the controls file was to have a way to easily select rules for a given "control" or requirement. Whether the rule gets into the product Benchmark depends on the `prodtype`.
A downside with this approach is that a single control item can seem to be cluttered with multiple similar rules that are applicable to different products. An alternative approach could be adding product specifiers in the controls file, but would drive power away from the rule itself and duplicate information in the rule and the controls file.

I agree that it is not simply to know what rules gets into the built profile, there used to be a log file for the rules processed in controls, but it got removed in https://github.com/ComplianceAsCode/content/pull/7716/files#diff-82be4fdbe0b5af5d652fdffd3d89415aa0d472a0ad7b6f674d8a1e54e765f4a8L131.

> For now this option seem not very explicit and I am looking for some mechanism to explicitly state that in the profile definition, so it is more obvious why this or this rule is included.
> So in that context I would gladly accept any suggestions and will be grateful of your advice on that.

Would you like other information available in the controls to get in the profile file as well? Like the `description`, `status` and `notes`?

Reply:
Hi @yuumasato thanks for the great explanation and detailed reply. In general I was looking more like an option to add on per rule basis, there should be no more than 5-6, if a certain rule in the control file is irrelevant for certain product. But that may make the control file really hard to read/support etc. So I tend to accept the policy if one wants to look of the actual profile for a product, go look compiled ones. :bow: 
My general plan then, as soon we at SUSE are ready with our current effort to cover targeted ANSSI profile, to loose the anssi_sle modification, as it will match 95% with the anssi control and work with the generic control file.
Once again thanks for the support on this :beers: